server/core/runtime/state_rm_step_runner_test.go

```diff
@@ -0,0 +1,61 @@
+package runtime
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/hashicorp/go-version"
+	. "github.com/petergtz/pegomock"
+	runtimematchers "github.com/runatlantis/atlantis/server/core/runtime/mocks/matchers"
+	"github.com/runatlantis/atlantis/server/core/terraform/mocks"
+	tfmatchers "github.com/runatlantis/atlantis/server/core/terraform/mocks/matchers"
+	"github.com/runatlantis/atlantis/server/events/command"
+	"github.com/runatlantis/atlantis/server/events/models"
+	"github.com/runatlantis/atlantis/server/logging"
+	. "github.com/runatlantis/atlantis/testing"
+)
+
+func TestStateRmStepRunner_Run_Success(t *testing.T) {
+	logger := logging.NewNoopLogger(t)
+	workspace := "default"
```

Comment:
Can we test with a non default workspace too ?

Reply:
[fix: state rm works on workspace](https://github.com/runatlantis/atlantis/pull/2880/commits/2ee0bc1501c2a385f2037440d7eef6cc2cbfcab5) 👍

Reply:
Nice!